site/docs/plugins/parse-mode.md

```diff
@@ -2,27 +2,134 @@
 
 This plugin provides a transformer for setting default `parse_mode`, and a middleware for hydrating `Context` with familiar `reply` variant methods - i.e. `replyWithHTML`, `replyWithMarkdown`, etc.
 
-## Usage
+## Usage (Using format)
```

Comment:
We need some sort of explanation what format actually use does and why people should use it (no escaping!) Also, we need a link to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#tagged_templates because people will think it's magic if they don't know the language well enough.

Reply:
@KnightNiwrem are you still interested in working on this?

Reply:
To be done in a different PR

Reply:
See #570